components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/utils/Constants.java

```diff
@@ -37,6 +42,11 @@ public class Constants {
                 add(PASSWORD);
                 add(USERNAME);
             }});
+    public static final Map<String, String> FIELD_VALIDATION_CONFIG_HANDLER_MAP =
```

Comment:
Whats the purpose of having the map here?

Reply:
We need to keep a map between fieldValidationConfiguration class against the field.

Reply:
We need this to do ?